packages/extism/extism.0.3.0/opam

```diff
@@ -0,0 +1,47 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Extism bindings"
+description: "Bindings to Extism, the universal plugin system"
+maintainer: ["Extism Authors <oss@extism.org>"]
+authors: ["Extism Authors <oss@extism.org>"]
+license: "BSD-3-Clause"
+tags: ["topics" "wasm" "plugin"]
+homepage: "https://github.com/extism/extism"
+doc: "https://github.com/extism/extism"
+bug-reports: "https://github.com/extism/extism/issues"
+depends: [
+  "ocaml" {>= "4.14.1"}
+  "dune" {>= "3.2"}
+  "ctypes" {>= "0.18.0"}
+  "ctypes-foreign" {>= "0.18.0"}
+  "bigstringaf" {>= "0.9.0"}
+  "ppx_yojson_conv" {>= "v0.15.0"}
+  "extism-manifest"
```

Comment:
```suggestion
  "extism-manifest" {= version}
```

maybe a good idea to add `{ = version }`
as if not, the CI is doing some funny thing
<img width="242" alt="Screen Shot 2023-05-23 at 17 18 41" src="https://github.com/ocaml/opam-repository/assets/2413116/bb0bb573-50b1-4c2f-8d71-640fc39e35da">
(https://toxis.caelum.ci.dev/github/ocaml/opam-repository/commit/7a04acb337ad7bba41167ccc5ff6e8bec1c1a8cd)

[you can specify `(extism-manifest (= :version))` in the dune-project file in achieving this](https://dune.readthedocs.io/en/stable/dune-files.html#:~:text=For%20example%2C%20to%20generate%20depends%3A%20%5B%20pkg%20%7B%20%3D%20version%20%7D%20%5D%2C%20use%20(depends%20(pkg%20(%3D%20%3Aversion))).)